src/com/projectkorra/projectkorra/airbending/combo/Twister.java

```diff
@@ -146,7 +147,8 @@ public void progress() {
 				continue;
 			}
 			final Vector forceDir = GeneralMethods.getDirection(entity.getLocation(), this.currentLoc.clone().add(0, height, 0));
-			entity.setVelocity(forceDir.clone().normalize().multiply(0.3));
+			GeneralMethods.setVelocity(this,entity, forceDir.clone().normalize().multiply(0.3));
+			
```

Comment:
We don't need the white-space after the line. Additionally, please change to
`GeneralMethods.setVelocity(this, entity, forceDir.clone().normalize().multiply(0.3));`